helper/resource/testing.go

```diff
@@ -538,11 +540,6 @@ func Test(t testing.T, c TestCase) {
 	}
 	log.SetOutput(logWriter)
 
-	// We require verbose mode so that the user knows what is going on.
-	if !testing.Verbose() && !c.IsUnitTest {
```

Comment:
I need to see is `IsUnitTest` is used anywhere now, worried too much was cut in V2 and there are no safeguards around accidentally running acceptance tests that cost money

Reply:
nvm this is fine